ipatests/test_cmdline/test_help.py

```diff
@@ -174,3 +174,18 @@ def test_multiline_description():
         api.Backend.cli.run(['trust-add', '-h'])
 
     assert unicode(api.Command.trust_add.doc).strip() in help_ctx.stdout
+
+
+def test_bz1428690():
+    """
+    Test for BZ#1428690 - ipa-backup does not create log file at /var/log/
+    :return: None
+    :raises: AssertionError if the test fails
+    """
+    with CLITestContext(exception=SystemExit) as ctx:
+        api.Backend.cli.run(['backup'])
```

Comment:
The command should be `ipa-backup`.

Reply:
And it means you cannot use here `api.Backend.cli.run()` because that function is only for running methods registered from plugins loaded into the client. The full list of methods is available in `api.Backend.Command`. `ipa-backup` is a separate script that is only available on the server side.